databases/week3/meal-share-db.sql

```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE IF NOT EXISTS `meal-share-db`;
+USE `meal-share-db`;
+DROP DATABASE IF EXISTS `meal-share-db`;
+
+DESCRIBE meal;
+DROP TABLE meal;
+CREATE TABLE meal(
+id int unsigned NOT NULL AUTO_INCREMENT,
+  title VARCHAR (255) NOT NULL,
+  description TEXT NULL DEFAULT NULL,
+  location  VARCHAR (255) NOT NULL,
+  time DATETIME NULL DEFAULT NULL,
+  max_reservation INT unsigned,
+  price DECIMAL,

```

Comment:
When using DECIMAL you should also specify the total number of digits and the number of digits after the decimal point, see here: https://www.mysqltutorial.org/mysql-decimal/ :)